source/features/unfinished-comments.tsx

```diff
@@ -29,7 +29,7 @@ function updateDocumentTitle(): void {
 
 	if (document.visibilityState === 'hidden' && hasDraftComments()) {
 		documentTitle = document.title;
-		document.title = '(Draft comment) ' + document.title;
+		document.title = '✏️ (unsent) ' + document.title;
```

Comment:
```suggestion
		document.title = '(✏️ Draft) ' + document.title;
```

Reply:
it's not technically a draft, it's just not yet sent

Reply:
> it's not technically a draft as intended in emails, it's just not yet sent


FYI some messager like Telegram also phrase it as "Draft". I would say "unsent" is pretty obscure here.

![IMG_20211207_110134](https://user-images.githubusercontent.com/44045911/144958526-df88d361-8854-49c2-b1c5-ec925989085b.jpg)

Reply:
```suggestion
		document.title = '✏️ (draft) ' + document.title;
```